.github/workflows/meetings.yml

```diff
@@ -25,8 +25,8 @@ jobs:
           agendaLabel: 'tc agenda'
           meetingLabels: 'meeting'
           # https://github.com/expressjs/discussions/issues/195#issuecomment-1973732769
-          # Starting on 2024-03-18 at 9pm UTC (2024-03-18T21:00:00.0Z) with a period of 2 weeks (P2W)
-          schedules: '2024-03-18T19:00:00.0Z/P2W'
+          # Starting on 2024-03-18 at 6pm UTC (2024-03-18T18:00:00.0Z) with a period of 2 weeks (P2W)
+          schedules: '2024-03-18T18:00:00.0Z/P2W'
```

Comment:
Let's add a `timeZone` option instead to the library? Otherwise it'll need changing again in November...

Reply:
Opened https://github.com/pkgjs/meet/pull/225.